config/jsonconfig/config.go

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
So we've been mostly doing comments using // on every line can we keep it consistent.  This also seems to be consistent with godocs best practices